src/tool/cppxlang/code_writers.h

```diff
@@ -1219,7 +1219,7 @@ namespace xlang
 
     private:
 
-        IReference<T>(IInspectable const& value) : IReference<T>(value.as<IReference<T>>())
+        IReference<T>(IXlangObject const& value) : IReference<T>(value.as<IReference<T>>())
```

Comment:
We haven't speced out boxing/nullable yet. Should we still have IReference here?

Reply:
It should be removed from the Foundation PR.